### ISSUE TITLE
Enhancement: Reusable lightning-spinner with text

### ISSUE THREAD

**Issue by sfdcschwabe** [LawrenceLoz/FormulaShare-DX#21]
During research I found this [video](https://www.youtube.com/watch?v=jzQU68TtY_c) which is very similar to your implemented approach with the lightning-spinner which is shown when you (de)activate rules.

But the difference with the above solution in the video is: You don't have to hardcode the subtitle of the spinner in your css.

**Comment by LawrenceLoz** [comment]
Yes agree this would be an improvement. Controlling the delay of the second message being shown if loading isn't fast enough in your new component would also simplify the logic in the places it's called.
